File: cash.c/cash.c
#include <cs50.h>
#include <stdio.h>
int coin (int n);
int main(void)
{
    int n;
    do
    {
        n=get_int("Number of coins: ");

    }while (n<=0);
    printf("%i\n", coin(n) );

}
int coin (int n)
{
    int contor=0;
    while (n)
    {
        if (n>=25)
        {
            n-=25;
            contor++;
        }
        else if (n>=10)
        {
            n-=10;
            contor++;
        }
        else if (n>=5)
        {
            n-=5;
            contor++;
        }
        else{
            n-=1;
            contor++;
        }
    }
    return contor;
}

File: sentimental-cash/cash.py
from cs50 import get_float
while True:
    change=get_float("Change: ")
    if change>=0:
      break
contor=0
while change!=0:
   if change>=0.25:
      change-=0.25
      contor+=1
   elif change>=0.10:
      change-=0.10
      contor+=1
   elif change>=0.05:
      change-=0.05
      contor+=1
   elif change>=0.01:
      change-=0.01
      contor+=1
   change=round(change,2)
print(contor)
